packages/client/src/app/pages/content-views/section-view/section-view.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit } from '@angular/core';
+import { ContentService } from '../../../services/content';
+import { ContentModel, SectionInfo } from '@pulp-fiction/models/content';
+import { Section } from '@pulp-fiction/models/sections';
+import { ActivatedRoute, Router } from '@angular/router';
+import * as v from 'voca';
```

Comment:
Importing the entire `voca` library just for their `slugify()` function strikes me as _super_ wasteful. I am making some assumptions about size though--what's the difference in compiled client size?

Voca's MIT-licensed--we could probably just lift their implementation without bringing the whole library along for the ride.